src/ImageBuilder/custom/New-AzImageBuilderTrigger.ps1

```diff
@@ -0,0 +1,173 @@
+
+# ----------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Code generated by Microsoft (R) AutoRest Code Generator.Changes may cause incorrect behavior and will be lost if the code
+# is regenerated.
+# ----------------------------------------------------------------------------------
+
+<#
+.Synopsis
+Create or update a trigger for the specified virtual machine image template
+.Description
+Create or update a trigger for the specified virtual machine image template
+.Example
+{{ Add code here }}
+.Example
+{{ Add code here }}
+
+.Outputs
+Microsoft.Azure.PowerShell.Cmdlets.ImageBuilder.Models.Api20220701.ITrigger
+.Link
+https://learn.microsoft.com/powershell/module/az.imagebuilder/new-azimagebuildertrigger
+#>
+function New-AzImageBuilderTrigger {
```

Comment:
can't tell what is this customization for, can you please explain

Reply:
README.md Line 79-81 
Hiding all the syntax called CreateExpanded, the command is incorrectly hidden.
I will fixed it.

Reply:
Done.